README.md

```diff
@@ -3,35 +3,94 @@
 </br>
 </br>
 
-<div align="center">
+<p align="center">
+  <img src="assets/logo.png" alt="xk6-disruptor" width="300"></a>
+  <br>
+  "Like Unit Testing, but for <strong>Reliability</strong>"
+  <br>
+</p>
+<p align="center">  
+</div>
 
-![logo](assets/logo.png)
+<blockquote align="center">
+⚠️ <strong>Important</strong> ⚠️ 
+<br>This project is still in its early stages and under active development.
+<br>The API is subject to change, and there may be bugs.
+<br>Use at your own risk. Thanks!
+</blockquote>
 
+This extension adds fault injection capabilities to [Grafana k6](https://github.com/grafana/k6). It implements the ideas of the Chaos Engineering discipline and enables Grafana k6 users to test their system's reliability under turbulent conditions.
 
-</div>
+## Why another Chaos Engineering tool?
 
-The xk6-disruptor is a [k6](https://k6.io) extension providing fault injection capabilities to test system's reliability under turbulent conditions. Think of it as "like unit testing, but for reliability". 
+Compared to other tools in the space, this one is purposely designed and built to provide the best experience for developers trying to make their systems more reliable:
 
-This project aims to aid developers in building reliable systems, implementing the goals of "Chaos Engineering" discipline in a k6 way - with the best developer experience as its primary objective. 
+- Everything as code.
+  - No need to mess with YAML or learn a new DSL.
```

Comment:
I like the "no need to learn a new DSL" but I don't feel comfortable with "mess with YAML" part. I find it a weak differentiation mostly considering we are in the Kubernetes ecosystem on which yaml is still prevalant. 
```suggestion
  - No need to learn a new DSL.
```